OpenGL/Constructs/VAO.cs

```diff
@@ -568,8 +587,8 @@ public void BindAttributes(ShaderProgram program)
         public delegate void DrawFunc();
         public delegate void DrawInstancedFunc(int count);
 
-        public DrawFunc Draw;
-        public DrawInstancedFunc DrawInstanced;
+        public DrawFunc Draw { get; private set; }
+        public DrawInstancedFunc DrawInstanced { get; private set; }
```

Comment:
It might be worth keeping these as public fields.  The first benefit is that it allows a user to provide a custom draw method if necessary.  The second benefit is that a property is expanded into a Get and Set method, which have extra calling overhead.  I would consider draw calls to be on the critical path when rendering, so any time we can get a bit more speed I would take that route.

Reply:
Darn, ya I failed to notice that Draw and DrawInstanced were now implementations of the interface.  It isn't a huge performance penalty, but it does exist.